docs/admin/install/slack_setup.md

```diff
@@ -6,8 +6,10 @@ These are the four distinct configuration values you will need to configure in `
 | ---------------------------- | ---------- | ------- |
 | `enable_slack`               | **Yes**    | False   |
```

Comment:
Note that line 3 states there are `four distinct configuration values` - probably just remove the `four`?

Reply:
Sure